schematools/contrib/django/factories.py

```diff
@@ -93,6 +93,26 @@ def handle_relation(
 
             if field._parent_table.has_parent_table:
                 kwargs["related_name"] = field._parent_table["originalID"]
+            else:
+                related_name = None
+                _, related_table_name = relation.split(":")
+                relations = {}
+                for table in dataset.tables:
+                    if table.id == related_table_name and table.relations:
+                        relations = table.relations
+                        break
+                if relations:
+                    for name, relation in relations.items():
```

Comment:
```suggestion
                try:
                    table = dataset.get_table_by_id(related_table_name)
                except ValueError:
                    pass
                else:
                    for name, relation in table.relations.items():
```

Als ik jouw code lees zou dit hetzelfde doen?

Reply:
Ik heb het aangepast dat get_table_by_id wordt gebruikt